src/containers/aspect/AspectDashboardContainer/AspectDashboardContainer.tsx

```diff
@@ -114,10 +116,17 @@ const AspectDashboardContainer: FC<AspectDashboardContainerProps> = ({
     }));
 
   const commentsPrivileges = aspect?.comments?.authorization?.myPrivileges ?? [];
+  const canDeleteComments = commentsPrivileges.includes(AuthorizationPrivilege.Delete) ?? false;
+  const canDeleteMessage = useCallback(
+    msgId =>
+      canDeleteComments || (isAuthenticated && (messages.find(x => x.id === msgId)?.author?.id === user?.id ?? false)),
```

Comment:
`(messages.find(x => x.id === msgId)?.author?.id === user?.id ?? false)` seems to be a good candidate for a function like `isAuthor(messages, user)`.